README.md

```diff
@@ -1,2 +1,27 @@
-# allennlp-simple-server-visualization
-static-dir files for a simple-server demo with ReactJS visualizations
+# Stand-alone demo with nice visualizations
+
+This is a pared-down version of the AllenNLP demo intended for use with a single model.  The basic
+idea here is that you copy the code in the `demo/` directory, modify two files to create a UI
+that's specific to your model, then run the demo using `npm`, hooking up to the AllenNLP simple
+server.
+
+For a more detailed explanation for how to make this work with your model, see the AllenNLP
+tutorial on visualizing your model (TODO - not yet written).  Here is a very brief outline of the
+steps involved:
+
+1. Get a simple text in / text out demo of your model running by following the [Predictor
+   tutorial in
+AllenNLP](https://github.com/allenai/allennlp/blob/master/tutorials/getting_started/making_predictions_and_creating_a_demo.md).
+ We'll be sending requests to this server using a separate UI, so once it works, just keep that
```

Comment:
I'd probably say something like

"We'll be using this server as the backend for our demo, so once it works..."

Reply:
Done.